### PR TITLE
You can now search the neon cocktails drinklist by taste tag

### PR DESCRIPTION
**Pull request by MadmanMartian**
## About The Pull Request   

Previously, you could only search for the drink name, and how strong it was. This didn't prove very useful in narrowing down what a customer would want, as the new sanity system wants things of a certain taste, not of a certain name.

## Why It's Good For The Game

As people come by to ask for bitter, or sweet, or spicy drinks, you can now search by that given descript.

## Changelog
:cl:
add: You can now search by taste type in the neon cocktails catalogue
/:cl:

### CODE REVIEW

**Review by TorinoFermic — APPROVED** [review]
I like this simple change

**Review by genessee596 — APPROVED** [review]
Delegate stamp of approval.

Based.

Has maintainer and delegate approval, so I'm gonna merge it.

**Pull request merged**
